backend/sirius-components-emf/src/main/java/org/eclipse/sirius/components/emf/services/ObjectService.java

```diff
@@ -47,6 +48,7 @@
  * @author hmarchadour
  */
 @Service
+@ConditionalOnMissingBean
```

Comment:
Same question here, if you need the EMF part of Sirius Components, why not use its object service?

Reply:
I think I begin to see what you mean... I'm trying to not depend on EMF. How can I avoid having the services & other Spring components of sirius-components-emf scanned? Marking them as @ConditionalOnMissingBean works but may not be the best answer.
However, I still believe that even if I want to depend on EMF, it's be legitimate for an app to be able to inject their own implementation (which can be achieved by annotating it with @Primary)